api/capi/src/nnstreamer-capi-single.c

```diff
@@ -518,6 +518,18 @@ ml_single_open_custom (ml_single_h * single, ml_single_preset * info)
       status = ML_ERROR_INVALID_PARAMETER;
       goto error;
     }
+  } else if (nnfw == ML_NNFW_TYPE_ARMNN) {
```

Comment:
Question: with armnn, is it possible to get the in/out tensors information from model?

Reply:
Yes, it is possible. However, this is dependent on the framework. 
With tensorflow-lite as backend, all information is available and no information needs to be passed. 
With caffe, input/output layer names, types and num_tensors are needed. Once provided, other information can be taken. Extra information provided from user can also be passed forward.